omnigan/blocks.py

```diff
@@ -85,7 +90,7 @@ def __init__(
             self.norm = LayerNorm(norm_dim)
         elif norm == "adain":
             self.norm = AdaptiveInstanceNorm2d(norm_dim)
-        elif norm == "spectral":
+        elif norm == "spectral" or norm.startswith("spectral_"):
```

Comment:
I'm not sure it's necessary to add the `or` condition here, since `norm="spectral_instance"` will have been change by `norm="instance"` by then.